core/src/interactive.rs

```diff
@@ -316,6 +321,8 @@ impl BroadcastMessage {
             AdvancementPolicySet { policy: AdvancementPolicy::DefendPoints } => format!("{} required players to defend on points", n?),
             KittySizeSet { size: Some(size) } => format!("{} set the number of cards in the bottom to {}", n?, size),
             KittySizeSet { size: None } => format!("{} set the number of cards in the bottom to default", n?),
+            FriendSelectionPolicySet { policy: FriendSelectionPolicy::Unrestricted} => "Friend can be selected by any card".to_string(),
+            FriendSelectionPolicySet { policy: FriendSelectionPolicy::HighestCardNotAllowed} => "Friend can not be selected by highest card".to_string(),
```

Comment:
nit

"{} disallowed the highest non-trump card, as well as trump cards, from being selected as a friend"

Reply:
trump card selection is not an option after my previous fix. change to: 

"{} disallowed the highest non-trump card from being selected as a friend"